.github/workflows/deploy-release.yml

```
@@ -21,6 +21,17 @@ jobs:
   do-release:
     if: github.repository_owner == 'hazelcast'
     runs-on: ubuntu-latest
+    services:
+      hazelcast:
+        image: hazelcast/hazelcast
```

This should be faster:
```suggestion
        image: hazelcast/hazelcast:latest-slim
```